manifests/kots-config.yaml

```diff
@@ -234,10 +236,18 @@ spec:
           hidden: true
           readonly: true
           when: &when_router_https_enabled >-
-            {{repl $customAccessModeSelected := (ConfigOptionEquals "accessToCartoModeK8s" "accessToCartoModeK8sCustom") -}}
+            {{ /* Kind of platforms */ -}}
             {{repl $isEmbeddedCluster := (ConfigOptionEquals "isEmbeddedCluster" "true") -}}
+            {{repl $isK8sFullyImplPlatform := (ConfigOptionEquals "isK8sFullyImplementedPlatform" "true") -}}
+            {{repl $isK8sNotFullyImplPlatform := (ConfigOptionEquals "isK8sNotFullyImplementedPlatform" "true") -}}
+            {{ /* Access mode */ -}}
+            {{repl $customAccessModeSelectedInFull := and $isK8sFullyImplPlatform (ConfigOptionEquals "accessToCartoModeK8s" "accessToCartoModeK8sCustom") -}}
+            {{repl $customAccessModeSelectedInNotFull := and $isK8sNotFullyImplPlatform (ConfigOptionEquals "accessToCartoModeK8sNotFullySupported" "accessToCartoModeK8sCustom") -}}
```

Comment:
This one will always be `true` no? 🤔

Reply:
🤔 I've been checking the code, and it makes sense to me as we'll be forcing the customer to select this one but it is "a decision made in the customer side" so IMHO we can't leave it like this 😄